src/main/java/com/abramchik/taskTwoCollections/hashMap/MyHashMapImpl.java

```diff
@@ -0,0 +1,228 @@
+package com.abramchik.taskTwoCollections.hashMap;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class MyHashMapImpl<K, V> implements MyHashMap<K, V> {
+
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+
+    private Node<K, V>[] hashTable;
+    private int size = 0;
+    private float threshold;
+
+
+    public MyHashMapImpl() {
+        hashTable = new Node[DEFAULT_INITIAL_CAPACITY];
+        threshold = DEFAULT_INITIAL_CAPACITY * DEFAULT_LOAD_FACTOR;
+    }
+
+    private int hash(K key) {
+        int hash = 31;
+        hash = hash * 17 * key.hashCode();
```

Comment:
Magic number, make it constant with understandable naming

Reply:
done